composer.json

```diff
@@ -11,7 +11,8 @@
     "ext-json": "*",
     "ext-libxml": "*",
     "ext-mbstring": "*",
-    "ampproject/amp-wp": "^2.0.10",
+    "ampproject/amp-toolbox": "^0.1",
+    "ampproject/amp-wp": "dev-develop",
```

Comment:
Wouldn't it be better to do `@stable` ?

Reply:
Wow you're fast!

See the PR description on why this is necessary at the moment.

In short: only AMP plugin 2.1+ will support Composer 2, but it is still in development (= no stable version to use yet)

Reply:
🤦‍♂️  I didn't read it before reviewing. I personally really hate not define versions like this. I understand why it is done, but I would like it changed once we can make as stable.